app/assets/stylesheets/articles.scss

```diff
@@ -1359,12 +1375,10 @@
         margin: 8px auto 0px;
         padding: 10px 0px;
         font-size: 0.93em;
-        color: darken($sky-blue, 17%);
-        color: var(--theme-color, darken($sky-blue, 17%));
-        background: white;
-        background: var(--theme-container-background, #fff);
-        border: 2px solid darken($sky-blue, 17%);
-        border: 2px solid var(--theme-color, darken($sky-blue, 17%));
+        @include themeable(color, theme-color, darken($sky-blue, 17%));
+        @include themeable(background, theme-container-background, #fff);
+        border: 2px solid;
+        @include themeable(border-color, theme-color, darken($sky-blue, 17%));
```

Comment:
The only thing that is themed here, is the border color, so it made sense to pull this out. There are fancier ways to do this with a mixin, but I opted to keep things simple.